app/com/github/pedrovgs/dotto/interpreter/DottoInterpreter.scala

```diff
@@ -0,0 +1,46 @@
+package com.github.pedrovgs.dotto.interpreter
+
+import akka.actor.ActorRef
+import cats.{Id, ~>}
+import com.github.pedrovgs.dotto.core.algebra.dotto._
+import com.github.pedrovgs.dotto.interpreter.actors.ShowMessageActor.ShowMessage
+import com.github.pedrovgs.dotto.interpreter.led.ledController._
+import com.github.pedrovgs.dotto.interpreter.morse.morseTranslator._
+import play.api.Logger
+
+/**
+  * Production interpreter implemented using Cats as core library.
+  */
+class DottoInterpreter(val showMessageActor: ActorRef) extends (Instruction ~> Id) {
```

Comment:
It's still unclear to me why you need actors for this problem and if you don't this is usually considered abuse. Stick to actors for concurrency or async issues where the problem is clearly needing them.